inc/class-licensing.php

```diff
@@ -427,7 +427,7 @@ public function getLicense( $license, $copyright_holder, $link, $title, $copyrig
 		} elseif ( $this->isSupportedType( $license ) ) {
 			$name = $this->getNameForLicense( $license );
 			$url  = $this->getUrlForLicense( $license );
-			if ( \Pressbooks\Utility\str_starts_with( $license, 'cc' ) && $license !== 'cc-zero' ) {
+			if ( \Pressbooks\Utility\str_starts_with( $license, 'cc' ) || \Pressbooks\Utility\str_starts_with( $license, 'ontario' ) && $license !== 'cc-zero' ) {
```

Comment:
`\Pressbooks\Utility\str_starts_with` comes from a time where PHP didn't have a built-in function to check that. We could replace them with PHP's built-in function `str_starts_with`.
```suggestion
			if ( str_starts_with( $license, 'cc' ) || str_starts_with( $license, 'ontario' ) && $license !== 'cc-zero' ) {
```